src/lib/statistics/statistics_objects/min_max_filter.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <iostream>
+#include <memory>
+#include <optional>
+
+#include "abstract_statistics_object.hpp"
+#include "all_type_variant.hpp"
+#include "types.hpp"
+
+namespace opossum {
+
+/**
+ *  Filter that stores a segment's minimum and maximum value
+*/
+template <typename T>
+class MinMaxFilter : public AbstractStatisticsObject {
```

Comment:
Naming and/or documentation, more for @Bouncner: What makes this a "filter"?

Reply:
I thought statistics objects used for chunk pruning carry the suffix "filter" short for "pruning filter". Right @Bouncner